src/components/Books.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const Books = () => (
+  <>
+    <div>
+      <span>
+        Action
+      </span>
+      <h2>
+        The Hunger Games
+      </h2>
+      <a href>
+        Suzanne Collins
+      </a>
```

Comment:
- [ ] The book title and Author + categories should be as prop data then you call the prop data.

![kkk](https://user-images.githubusercontent.com/61562626/140792178-05b0478a-2a21-4b4b-ad37-a39ef5dedb19.jpeg)